config/locales/en/images/crop.yml

```diff
@@ -3,3 +3,4 @@ en:
     crop:
       title: "Crop image for ‘%{title}’"
       title_modal: "Crop image"
+      title_exact_dimensions: "Image does not need to be cropped"
```

Comment:
This one is a little jarring for when you click edit image and then are presented with "Image does not need to be cropped" as it feels like the title and what you requested don't match.

![image-crop](https://user-images.githubusercontent.com/282717/57451099-2c740400-7258-11e9-909e-75e2d937dd96.gif)

Reply:
I think this is acceptable for a quick improvement. 
If the user wants to edit the image they need reminding that they can't crop. They might be trying to adjust the crop, but in this situation they need to be told they can't.

The alternative would be to have separate flow for exact size images to skip this screen when edited, but the user wouldn't know why.
Or to go back to having separate links for edit crop and image title.

The prior content, Crop Image, was worse as it told users to do something that they couldn't do.